lab_1/main.py

```diff
@@ -11,26 +11,63 @@ def tokenize(text: str) -> list or None:
     :param text: a text
     :return: a list of lower-cased tokens without punctuation
     """
-    pass
 
+    if isinstance(text, str):  # text является экземпляром классом str
```

Comment:
Let's write checks with None at the beginning
If the arguments are incorrect, we return None

Reply:
done